src/lib/controllers/Client/index.js

```diff
@@ -0,0 +1,14 @@
+class ClientController {
+  static disconnectionHandler() {
+    console.log('Client close');
+    // TODO
+  }
+
+  static connectHandler(socket, moduleContainer) {
+    console.log('Client open');
+    socket.on('bridge-data', moduleContainer.handle);
+    socket.on('disconnect', this.disconnectionHandler);
```

Comment:
Same here, string in const

Reply:
`this. disconnectionHandler ` works in a `static` context ? 🤔

Reply:
According to the following test yes:
```js 
  it('should call on function', () => {
    ClientController.connectHandler(socket, moduleController);
    expect(socket.on).toBeCalledWith('disconnect', ClientController.disconnectionHandler);
  });
```

Reply:
Actually, it can't be. `this` relies on an object. But you don't create any object 😛